sql/plan/sort.go

```diff
@@ -397,9 +399,9 @@ type topRowsIter struct {
 	idx           int
 }
 
-func newTopRowsIter(s sql.SortFields, limit int64, calcFoundRows bool, child sql.RowIter) *topRowsIter {
+func newTopRowsIter(s sql.SortFields, limit int64, calcFoundRows bool, child sql.RowIter, childLen int) *topRowsIter {
```

Comment:
Nit: rename `childLen` to `childSchemaLen`?

Reply:
added